### PR TITLE
Bump debian-* base image versions to latest

### PR DESCRIPTION
**Pull request by tallclair**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Follow up to https://github.com/kubernetes/kubernetes/pull/73493 to use the new images.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @ixdy 
/priority critical-urgent
/sig release

### CODE REVIEW

File: build/root/WORKSPACE [hunk 1]
```diff
@@ -73,23 +73,23 @@ docker_pull(
     digest = "sha256:86176bc8ccdc4d8ea7fbf6ba4b57fcefc2cb61ff7413114630940474ff9bf751",
```
Comment (ixdy) [inline build/root/WORKSPACE]: the digests are what actually matter here.

Comment (tallclair) [reply build/root/WORKSPACE]: oops, thanks. Done.

**Comment by rtheis** [comment]
Is this (along with https://github.com/kubernetes/kubernetes/pull/73493) something that the community would consider cherry-picking to 1.11, 1.12 and 1.13?

**Pull request merged**